#!/usr/bin/env python
PACKAGE = "wanji_driver"
NODE_NAME = "wanji_driver_node"
PARAMS_NAME = "WanjiNode"

from math import pi
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("time_offset", double_t,  0, "A manually calibrated offset (in seconds) to add to the timestamp before publication of a message.",
        0.0, -1.0, 1.0)

exit(gen.generate(PACKAGE, NODE_NAME, PARAMS_NAME))
